CONTRIBUTING.md

```diff
@@ -97,6 +97,32 @@ and other collections of tests for different components.
 When writing a new test, there should usually be a test of
 similar functionality already written and related tests should
 be added nearby.
-When in doubt, feel free to ask.
 
-TODO: describe some details about fixtures, etc.
+The fixtures live in `jupyterhub/tests/conftest.py`. There are
+fixtures that can be used for JupyterHub components, such as:
+
+- `app`
+- `auth_state_enabled`
+- `db`
+- `io_loop`
```

Comment:
There are some fixtures (``user``, ``admin_user`` and ``event_loop``) that I believe were added later and aren't included in the ``conftest.py`` docstring, but might be worth to mention them here. What do you think?

Reply:
Good idea @GeorgianaElena ! Done!

Reply:
Yaay! Thanks @kinow :heart:. One more little thing: how do you feel about adding a short (few words) description of each of the fixtures? I believe this would be a great addition for people contributing for the first time.

Reply:
Sure, I started with a short text for the app, but then decided to start by just copying the docstrings.

Let me have a go at it.

Reply:
Not sure if the best wording, but happy to update it if necessary :+1: